runJob.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
To be consistent with the current "architecture", I would rename `slurmTools.sh` to `schedulerTools.sh` and add this functionality to a new function`run_script()` in that file. The check whether to use SLURM or simply launch a shell can be made therein. This should not be a big change, it's simply moving things around and naming.

Reply:
If you nevertheless decide against this, there are a couple of comments that I would make regarding this file which are oblivious otherwise, so let me know.

Reply:
That's a good idea, I'll try that!